src/main/java/uk/gov/pay/ledger/agreement/entity/AgreementsFactory.java

```diff
@@ -32,4 +35,10 @@ public PaymentInstrumentEntity createPaymentInstrument(EventDigest eventDigest)
         entity.setEventCount(eventDigest.getEventCount());
         return entity;
     }
+
+    private AgreementStatus getStatus(EventDigest eventDigest) {
```

Comment:
I think `AgreementsFactoryTest` can be updated to remove the `status` from the event payload, as it will no longer be used to resolve the status.

Reply:
We can also remove the "status" from `QueueMessageReceiverIT` tests for agreement event tests